plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -320,19 +322,6 @@ SecRule REQUEST_FILENAME "@rx /remote\.php/dav/files/[^/]+/$" \
         "t:none,\
         ctl:ruleRemoveById=950100"
 
-# Syncing files with Nextcloud desktop app
-SecRule REQUEST_FILENAME "@contains /remote.php/dav/files/" \
-    "id:9508174,\
-    phase:3,\
-    pass,\
-    t:none,\
-    nolog,\
-    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
-    chain"
-    SecRule RESPONSE_STATUS "@streq 500" \
-        "t:none,\
-        ctl:ruleRemoveById=950100"
-
```

Comment:
My mistake, I thought this was ``ctl:ruleRemoveById=950000-950999,\`` when it's actually ``ctl:ruleRemoveById=951000-951999,\`` https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/pull/40/files#diff-b927017cb1fee736fcfe55bdef8b2ca3f4966a9896ea93d3e1e6f29487303be0R141

In that case, removing rules ``9508173`` and or ``9508174`` will introduce a regression. ``9508173`` is meant to match ``/remote.php/dav/files/username/`` and ``9508174`` is meant to match ``/remote.php/dav/files/path/to/example.txt``.

You could bring back rule ``9508174`` and modify ``RESPONSE_STATUS`` to match ``^50(?:3|2|0)$`` and remove ``9508173`` or leave as is.

Reply:
Well spotted, thanks. I've pushed a change that should work for both rules.

Reply:
You also need to modify the chained rule to accept status codes of 500 ```^50(?:3|2|0)$```

Reply:
Ups, missed that. Thanks.